### PR TITLE
assistant bulder template help

### PR DESCRIPTION
**Pull request by fontanierh**
## Description
Screenshot 2024-03-28 at 10 05 10

When there is a templateId, the assistant builder "preview" drawer is automatically opened, has 2 tabs, and automatically goes to the new "template" tab that displays the template's "instructions help".
If there is no template, behaviour is unchanged.

### CODE REVIEW

File: front/components/assistant_builder/AssistantBuilder.tsx [hunk 1]
```diff
@@ -523,23 +555,77 @@ export default function AssistantBuilder({
...
+                  )}
+                >
+                  <TryAssistant
+                    owner={owner}
+                    assistant={draftAssistant}
+                    conversationFading={isFading}
+                  />
+                </div>
+              ) : (
+                <ReactMarkdown
```
Comment (fontanierh) [inline front/components/assistant_builder/AssistantBuilder.tsx:586]: will maybe try to DRY this with message rendering later on @flvndvd

Comment (flvndvd) [reply front/components/assistant_builder/AssistantBuilder.tsx:586]: Can we at least extract it in a component in this file?

**Review by flvndvd — APPROVED** [review]
LGTM, left one nit + if you can increase the right padding to 32px.

File: front/components/assistant_builder/AssistantBuilder.tsx (hunk 1 above)
Comment (fontanierh) [reply front/components/assistant_builder/AssistantBuilder.tsx:586]: WIll do separately, I think we want markdown component that can already render everything so we can use in both chat and here

**Pull request merged**